bugbug/tools/code_review.py

```diff
@@ -942,6 +1023,55 @@ def run(self, patch: Patch) -> list[InlineComment] | None:
 
         return list(generate_processed_output(raw_output, patch_set))
 
+    def _get_comment_examples(self, patch):
+        comment_examples = []
+
+        if self.review_comments_db:
+            # TODO: use a smarter search to limit the number of comments and
+            # diversify the examples (from different hunks, files, etc.).
+            comment_examples = self.review_comments_db.find_similar_patch_comments(
+                patch, limit=10
+            )
```

Comment:
Thought: we might want to mix comments written by developers and comments written by the bot (and accepted by developers). Perhaps it could be a number we can configure manually. This is a follow-up of course.

Reply:
I like the idea. I filed a follow-up issue: https://github.com/mozilla/bugbug/issues/4320.